Firestore/core/src/firebase/firestore/remote/CMakeLists.txt

```diff
@@ -51,8 +51,8 @@ add_custom_command(
         --output_header=grpc_root_certificates_generated.h
         --output_source=grpc_root_certificates_generated.cc
         --cpp_namespace=firebase::firestore::remote
-        --array=grpc_root_certificates
-        --array_size=grpc_root_certificates_size
+        --array=grpc_root_certificates_generated_data
```

Comment:
@zxu123 Looking at your CL, this looks like the correct names, but please point out if I got this wrong.

Reply:
Confirmed. For reference, a sample build can be found at go/rigbr, check Output Files tab.

Reply:
Thanks.